### ISSUE TITLE
A11y Tweaks in base-slider.plugin.js

### ISSUE THREAD

**Issue by janobi** [shopware/platform#10787]
### Technical TODO

Since different plugins can add different additional selectable elements to slider contents (esp. product boxes), there should be an easy, standardized way to extend the list of selectable element types used in _initAccessibilityTweaks of base-slider.plugin.js ([code line](https://github.com/shopware/shopware/blob/v6.7.0.0/src/Storefront/Resources/app/storefront/src/plugin/slider/base-slider.plugin.js#L182)) to e.g. select or input elements.
If that seems too much of a hassle, the selector should be updated to include at least those two elements as well.

**Comment by janobi** [comment]
I should note that just overriding that plugin ist not a great option since you'd have to override the other plugins extending the BaseSliderPlugin class as well.
